internal/docstore/mongodocstore/codec.go

```diff
@@ -110,7 +110,15 @@ func (e *mapEncoder) MapKey(k string) {
 ////////////////////////////////////////////////////////////////
 
 // decodeDoc decodes m into ddoc.
-func decodeDoc(m map[string]interface{}, ddoc driver.Document) error {
+func decodeDoc(m map[string]interface{}, ddoc driver.Document, idField string) error {
+	switch idField {
+	case mongoIDField: // do nothing
+	case "": // user documents have no ID field
```

Comment:
The comment should be something like `user uses idFunc instead`, since if there is no ID field and ID func it will use the default ID field right?

Reply:
Done.